roboteam_ai/src/control/positionControllers/ForcePosControl.cpp

```diff
@@ -47,11 +48,13 @@ Vector2 ForcePosControl::calculateForces(const RobotPtr &robot, const Vector2 &t
             force += ControlUtils::calculateForce(Vector2(-1.0, -1.0) / robot->pos, FORCE_WEIGHT_FIELD_SIDES, 99.9);
         }
     }
-
+    force = (force.length() > 3.0) ? force.stretchToLength(3.0) : force;
     return force;
 }
 
 PosVelAngle ForcePosControl::calculateForcePosVelAngle(const PosController::RobotPtr& robot, const Vector2 &targetPos) {
+    BasicPosControl basicPosControl;
+    return basicPosControl.getPosVelAngle(robot, targetPos);
     double forceRadius;
```

Comment:
lines that are never hit here.

Reply:
force will be removed in numtrees pr.

Reply:
So, then this is on development in the mean time? I really do not like this.